rs/registry/canister/src/mutations/subnet.rs

```diff
@@ -262,35 +262,34 @@ impl Registry {
         }
     }
 
-    /// Get the list of subnets that can sign for a given MasterPublicKeyId.
-    pub fn get_chain_key_signing_subnet_list(
+    /// Get the list of subnets that are enabled for a given MasterPublicKeyId.
+    pub fn get_chain_key_enabled_subnet_list(
         &self,
         key_id: &MasterPublicKeyId,
-    ) -> Option<ChainKeySigningSubnetList> {
-        let chain_key_signing_subnet_list_key_id = make_chain_key_signing_subnet_list_key(key_id);
+    ) -> Option<ChainKeyEnabledSubnetList> {
+        let chain_key_enabled_subnet_list_key_id = make_chain_key_enabled_subnet_list_key(key_id);
         self.get(
-            chain_key_signing_subnet_list_key_id.as_bytes(),
+            chain_key_enabled_subnet_list_key_id.as_bytes(),
             self.latest_version(),
         )
         .map(|registry_value| {
-            ChainKeySigningSubnetList::decode(registry_value.value.as_slice()).unwrap()
+            ChainKeyEnabledSubnetList::decode(registry_value.value.as_slice()).unwrap()
         })
     }
 
-    /// Create the mutations that disable subnet signing for a single subnet
-    /// and set of MasterPublicKeyId's.
-    pub fn mutations_to_disable_subnet_signing(
+    /// Create the mutations that disable set of chain keys for a single subnet.
+    pub fn mutations_to_disable_subnet_chain_key(
         &self,
         subnet_id: SubnetId,
-        chain_key_signing_disable: &Vec<MasterPublicKeyId>,
+        chain_key_disable: &Vec<MasterPublicKeyId>,
     ) -> Vec<RegistryMutation> {
         let mut mutations = vec![];
-        for chain_key_id in chain_key_signing_disable {
+        for chain_key_id in chain_key_disable {
             let mut chain_key_signing_list_for_key = self
-                .get_chain_key_signing_subnet_list(chain_key_id)
+                .get_chain_key_enabled_subnet_list(chain_key_id)
                 .unwrap_or_default();
 
-            // If this subnet does not sign for that key, do nothing.
+            // If this subnet is already disabled for that key, do nothing.
```

Comment:
```suggestion
            // If that key is already disabled on this subnet, do nothing.
```

Is this also correct? (Choose whichever variant fits better)

Reply:
Yes, that's [better](https://github.com/dfinity/ic/pull/4819/commits/87a8d0c0694338334fb8a36e2c13ef4ebb7bed26), thanks!